Gopkg.toml

```diff
@@ -37,8 +37,10 @@
   version = "~1.3.0"
 
 [[constraint]]
-  version = "~0.2.0"
+  #version = "~0.2.0"
```

Comment:
I'm assuming we don't want these changes in the PR.

Reply:
This depends on the bundle lib PR so will update once we get a release of bundle lib